lib/perl/Genome/Config/AnalysisProject/Command/CompareInstrumentData.pm

```diff
@@ -0,0 +1,119 @@
+package Genome::Config::AnalysisProject::Command::CompareInstrumentData;
+
+use strict;
+use warnings;
+
+use Genome;
+
+use Set::Scalar;
+use Data::Dumper;
+
+class Genome::Config::AnalysisProject::Command::CompareInstrumentData {
+    is => ['Command::V2'],
+    has => [
+        first_analysis_project => {
+            is => 'Genome::Config::AnalysisProject',
+            doc => 'The first Analysis Project to compare Instrument Data.',
+            shell_args_position => 1,
+        },
+        second_analysis_project => {
+            is => 'Genome::Config::AnalysisProject',
+            doc => 'The second Analysis Project to compare Instrument Data.',
+            shell_args_position => 2,
+        },
+    ],
+    has_optional => [
+        list_intersection => {
```

Comment:
Alternatively, this could be replaced with a generic `list_details` option that then determines whether or not the `list_set` calls are executed.

```
if ($self->list_details && $set_x->size > 0) {
    $self->list_set($set_x);
}
```
etc.